content/docs/install/pre-release.md

```diff
@@ -10,16 +10,9 @@ releases, you can install it from our code repository GitHub.
 > (on Python 3.6+) to encapsulate your local environment.
 
 ```dvc
-$ pip install gitpython
-$ pip install git+https://github.com/iterative/dvc
+$ pip install --upgrade --pre dvc
 ```
 
-> `gitpython` allows the installation process to generate a DVC version using
-> the current Git commit hash. This lets us to distinguish official DVC releases
-> (e.g. `0.64.3`) from a development version (e.g. `0.64.3-9c7381`). For more
-> information on our versioning convention, refer to
-> [Components of DVC version](/doc/command-reference/version#components-of-dvc-version).
-
```

Comment:
Is this change correct? Not sure who to check with exactly so cc @efiop, @skshetry, and @dmpetrov 

I tried it just like this with and without gitpython and dvc version printed `DVC version: 1.0.0a0`.

Thanks

Reply:
No, this is not exactly correct. Since we now have dev and actual pre-release we need to describe how to install both. Or keep only the way it was before since I'm not sure we are going to do that many official pre-releases.

Reply:
Reverted in eee573fc. I think no need to describe both since it seems the pip option will be rarely used and we can always announce it on the blog.